packages/web-components/fast-foundation/src/dialog/README.md

```diff
@@ -18,4 +19,14 @@ import { DialogStyles as styles } from "./dialog.styles";
     styles,
 })
 export class FASTDialog extends Dialog {}
+```
+
+## Usage
+
+```html
+<fast-dialog id="simpleDialog" aria-label="Simple modal dialog" modal="true">
```

Comment:
Is there a way to embed fixtures into these documents? I worry about these getting out of sync and it would be awesome to just reference a fixture.

Reply:
I'm sure we could copy them over somehow, but I'd want to scrub our fixtures and be sure they are always at a high quality. I lean a bit towards having these be a bit curated to start. This is even initial as I'm going to do a follow-up to try and have live-code examples as part of the readme.

Reply:
We could have our build process copy the fixture content into a designated location in the markdown file.